terraform/stacks/main/outputs.tf

```diff
@@ -404,6 +404,18 @@ output "s3_gateway_endpoint_cidr_2" {
   value = data.aws_prefix_list.s3_gw_cidrs.cidr_blocks[1]
 }
 
+output "s3_gateway_endpoint_cidr_3" {
+  value = data.aws_prefix_list.s3_gw_cidrs.cidr_blocks[2]
+}
+
+output "s3_gateway_endpoint_cidr_4" {
+  value = data.aws_prefix_list.s3_gw_cidrs.cidr_blocks[3]
+}
+
+output "s3_gateway_endpoint_cidr_5" {
+  value = data.aws_prefix_list.s3_gw_cidrs.cidr_blocks[4]
+}
+
```

Comment:
We should replace all of the `s3_gateway_endpoint_cidr_*` blocks with:

```
output "s3_gateway_endpoint_cidrs" {
    value = data.aws_prefix_list.s3_gw_cidrs.cidr_blocks[*]
}
```

Otherwise, we'll still hardcoding an expectation of how many IPs there will be for the gateway endpoint and could run into the same problem again